### PR TITLE
Beginning the 1.27 release notes

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

File: pages/k8s/supported-versions.md
Comment (addyess) [inline pages/k8s/supported-versions.md:72]:
```suggestion
publisher: Canonical
```

File: pages/k8s/1.27/components.md
Comment (evilnick) [inline pages/k8s/1.27/components.md:99]: the line break here breaks the table - is this a problem in the generator script or did you manually add it?
image

File: pages/k8s/release-notes.md
Comment (evilnick) [inline pages/k8s/release-notes.md:97]: can we lose this from the template - it breaks the webteam's parser if there isn't a blank line after HTML elements (which of course, makes the ignore command not ignore the bit it is supposed to

File: pages/k8s/1.27/release-notes.md
Comment (evilnick) [inline pages/k8s/1.27/release-notes.md:96]: ditto

File: pages/k8s/1.27/components.md
Comment (addyess) [reply pages/k8s/1.27/components.md:99]: Looks like there could be a LF in the description/summary field of ceph-csi which generated this... Lemme update the generate to strip line `LF`s

File: pages/k8s/release-notes.md
Comment (addyess) [reply pages/k8s/release-notes.md:97]: we can either drop it from the templates and drop it from the rendered but it will make the woke checks fail on the repo for just this PR.  Afterwards it should be okay

Comment (addyess) [reply pages/k8s/release-notes.md:97]: i could put a blanket ignore of the word `master` on this repo... we can just stop checking for it

File: pages/k8s/release-notes.md
Comment (evilnick) [inline pages/k8s/release-notes.md:98]: I just realised this also fails - need to leave blank lines otherwise it gets rendered

File: pages/k8s/1.27/upgrading.md
Comment (kwmonroe) [inline pages/k8s/1.27/upgrading.md:1]: This page is pretty empty, but the [current 1.26 version](https://ubuntu.com/kubernetes/docs/1.26/upgrading) isn't.  @evilnick shouldn't there be meat in here?

**Review by kwmonroe — COMMENTED** [review]
Made some tweaks.  I'm confused about our `index.md` and `1.27/upgrading.md` pages, but otherwise this looks great.

File: pages/k8s/1.27/upgrading.md
Comment (evilnick) [reply pages/k8s/1.27/upgrading.md:1]: yes, i hve it in a different branch - i guess it is easiest to add to this PR

File: index.md
Comment (evilnick) [reply index.md:1]: it's lame. It only really exists because tim insisted we keep it - it doesn't get used anywhere other than our local previews

**Review by kwmonroe — APPROVED** [review]
@evilnick should get the final say, but this lgtm!

**Pull request merged**
